web-ui/src/main/resources/catalog/views/default/module.js

```diff
@@ -50,7 +50,8 @@
         params: {
           sortBy: 'popularity',
           from: 1,
-          to: 9
+          to: 9,
+          _retired: 'false'
```

Comment:
In other places this filter is `'false or true'`. I've used a string here for consistency.